spec/RateLimit.spec.js

```diff
@@ -335,6 +335,99 @@ describe('rate limit', () => {
     await Parse.Cloud.run('test2');
   });
 
+  describe('zone', () => {
+    const middlewares = require('../lib/middlewares');
+    it('can use global zone', async () => {
+      await reconfigureServer({
+        rateLimit: {
+          requestPath: '*',
+          requestTimeWindow: 10000,
+          requestCount: 1,
+          errorResponseMessage: 'Too many requests',
+          includeInternalRequests: true,
+          zone: 'global',
```

Comment:
Instead of a string, could you define an object that one can use and add the docs there:
```
/**
 * ...
 *
 * @memberof ...
 *
 * @typedef {String} RateLimitZone
 * @property {String} global Rate limit based on the number of requests made by all users.
 * ...
 */
static get RateLimitZone() {
  return Object.freeze({
    global: 'global',
    ...
  });
}
```